osmo_jupyter/plot_test.py

```diff
@@ -0,0 +1,65 @@
+import pandas as pd
+import pytest
+
+import osmo_jupyter.plot as module
+
+
+class TestGetColorsWithPattern:
+    def test_basic_case(self):
+        assert module._get_colors_with_pattern('*_mean') == ['r_mean', 'g_mean', 'b_mean']
+
+    def test_missing_asterisk__blows_up_helpfully(self):
+        with pytest.raises(ValueError, match=r'does not contain an asterisk'):
+            module._get_colors_with_pattern('lololololol')
+
+
+class TestGetRgbColumns:
+    def test_basic_case(self):
+        orig_df = pd.DataFrame({
+            'r_abcd': [1, 2, 3],
+            'g_abcd': [4, 5, 6],
+            'b_abcd': [7, 8, 9],
+            'other': [10, 11, 12]
+        })
+
+        actual = module.get_rgb_columns(orig_df, '*_abcd')
+
+        expected = pd.DataFrame({
+            'r': orig_df['r_abcd'],
+            'g': orig_df['g_abcd'],
+            'b': orig_df['b_abcd'],
+        })
+
+        pd.testing.assert_frame_equal(actual, expected)
+
+
+class TestAddRgbColumns:
+    rgb_df = pd.DataFrame({
```

Comment:
nit: maybe change to `rgb_abcd_df`

Reply:
also might be slightly clearer if you used a real term, like `*_mean` and `rgb_means_df`

Reply:
I changed it; didn't like the idea of implying a particular purpose of the `rgb_df` but I changed it to `rgb_df_to_add`